### DISCUSSION TITLE
AI無人機運送血漿

### DISCUSSION THREAD

**Discussion by Timmy110403540** [1101-GS3073-C/Python-Community-GS3073-C#122] [General]
雖然現在的大規模戰爭已漸消失，但不免有些區域性的衝突。
美國和 AI 廠商 Near Earth Autonomy 與 國防廠商 L3Harris 合作， 開發名為 FVR-90 的定翼垂直起降無人機。
它有著特殊的儲存箱，以及計算降落地區中的地形，並依據地形判斷降落時機。
當士兵們受傷時，透過無人機來運送血漿，支援前線的戰士們。
而現在美國遇到的困境-為如何支援如在海上的戰爭，可能需要一段時間來解決此問題。

新聞來源:https://technews.tw/2021/11/16/us-army-testing-ai-drones-for-delivering-whole-blood-to-battlefield/
##當然，減少戰爭，才是大家所希望的!

**Comment by minniewei** [comment]
這種科技如果用在山難或其他人員救助的場合我覺得會很有用處，但希望不要用在戰爭上。

**Reply by cobb110202008** [reply]
真的，如果用在戰爭上，感覺戰爭會拖的更久，對人民造成的傷害可能更多。

**Comment by pocper1** [comment]
無人機翼展4.5公尺，肉眼感覺很容易被發現，搞不好還沒送到目的地就被打下來了...

**Comment by awehao** [comment]
如果用在醫療用途，而非軍事用途，感覺會更加造福世界

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by lukiea** [comment]
我認為，在人類的歷史中，我們總是盡了一切努力想搞懂血漿。血漿的出現，必將帶領人類走向更高的巔峰。血漿必定會成為未來世界的新標準。

**Comment by larry40305** [comment]
而這些並不是完全重要，更加重要的問題是，看看別人，再想想自己，會發現問題的核心其實就在你身旁。AI無人機運送血漿對我來說有著舉足輕重的地位，必須要嚴肅認真的看待。

**Comment by boson13579** [comment]
希望FVR-90能盡量減少在戰場上應用的需求，轉而投入在救援天災的傷者上。
